cmd/epithet-oidc-plugin/epithet-oidc-plugin.go

```diff
@@ -35,6 +43,28 @@ func main() {
 	}
 }
 
+func logging(cmd *cobra.Command, args []string) {
+	if logPath != "" {
```

Comment:
How do you log to stdout?

Reply:
The other epithet binaries all log to stdout, and [use -vv style verbosity config](https://github.com/epithet-ssh/epithet/blob/master/cmd/epithet-agent/epithet-agent.go#L37-L47)

Reply:
I don't think we can log to stdout anymore, since state will be sent to stdout.

Reply:
Oh, duh!